src/network/errors.ts

```diff
@@ -2,35 +2,28 @@ import { ErrorPolykey, sysexits } from '../errors';
 
 class ErrorNetwork extends ErrorPolykey {}
 
-class ErrorForwardProxy extends ErrorNetwork {}
+class ErrorProxy extends ErrorNetwork {}
 
-class ErrorForwardProxyNotRunning extends ErrorForwardProxy {
-  description = 'ForwardProxy is not running';
+class ErrorProxyNotRunning extends ErrorProxy {
+  description = 'Proxy is not running';
   exitCode = sysexits.USAGE;
 }
 
-class ErrorForwardProxyInvalidUrl extends ErrorForwardProxy {
+class ErrorProxyInvalidUrl extends ErrorProxy {
   description = 'Invalid target host used for HTTP connect proxy';
   exitCode = sysexits.PROTOCOL;
 }
 
-class ErrorForwardProxyMissingNodeId extends ErrorForwardProxy {
+class ErrorProxyMissingNodeId extends ErrorProxy {
   description = 'Node ID query parameter is required for HTTP connect proxy';
   exitCode = sysexits.PROTOCOL;
 }
 
-class ErrorForwardProxyAuth extends ErrorForwardProxy {
+class ErrorProxyAuth extends ErrorProxy {
```

Comment:
I suggest that these should be:

```
ErrorProxyConnect...
```

The `Connect` error just tells us that this is part of the HTTP Connect protocol.

These exceptions are not caught in `src` atm, so it's a simple change here.